src/contexts/CommentsContext/commentsContextMessages.js

```diff
@@ -47,6 +47,8 @@ function beginListening(dispatch, diffDispatch) {
         diffDispatch(addContents(fixedUpForDiff));
         dispatch(updateCommentsFromVersions(commentDetails));
         break;
+      case DEMO_EVENT:
```

Comment:
Don't understand how this works. I thought something would live inside the account login and send along the demo quick add stuff to a listener if it was in the response.